events/events.go

```diff
@@ -0,0 +1,142 @@
+package events
+
+import (
+	"github.com/spacemeshos/go-spacemesh/log"
+	"github.com/spacemeshos/go-spacemesh/types"
+)
+
+const (
+	NewBlock channelId = 1 + iota
```

Comment:
I'd move this and the methods under publisher to a different file

Reply:
or atleast put the together instead of having the consts at the top and methods at bottom